packages/aws-cdk/README.md

```diff
@@ -372,6 +372,44 @@ For this reason, only use it for development purposes.
 **⚠ Note #2**: This command is considered experimental,
 and might have breaking changes in the future.
 
+### `cdk watch`
+
+The `watch` command is similar to `deploy`,
+but instead of being a one-shot operation,
+the command continuously monitors the files of the project,
+and triggers a `cdk deploy` whenever it detects any changes:
+
+```console
+$ cdk watch ExampleStack
+Detected change to 'lambda-code/index.js' (type: change). Triggering 'cdk deploy'
+ExampleStack: deploying...
+
+ ✅  ExampleStack
+
+^C
+```
+
+To end a `cdk watch` session, interrupt the process by pressing Ctrl+C.
+
+What files are observed is determined by the `"watch"` setting in your `cdk.json` file.
+It has two sub-keys, `"include"` and `"exclude"`, each which can be either a single string, or an array of strings.
+Each entry is interpreted as a path relative to the location of the `cdk.json` file.
+Globs, both `*` and `**`, are allowed to be used.
+Example:
+
+```json
+{
+  "app": "mvn -e -q compile exec:java",
+  "watch": {
+    "include": "src/main/**",
+    "exclude": "target/*"
+  }
+}
+```
+
+Note that `watch` by default uses hotswap deployments, and disables CloudFormation rollback.
```

Comment:
Mention how this interacts with "build"

Reply:
Same response as https://github.com/aws/aws-cdk/pull/17240#discussion_r740621652.